src/Microsoft.NetCore.Analyzers/Core/Security/SystemSecurityCryptographyResources.resx

```diff
@@ -405,4 +405,13 @@
   <data name="DoNotAddArchiveItemPathToTheTargetFileSystemPathMessage" xml:space="preserve">
     <value>When creating path for '{0} in method {1}' from relative archive item path to extract file and the source is an untrusted zip archive, make sure to sanitize relative archive item path '{2} in method {3}'</value>
   </data>
+  <data name="UseAutoValidateAntiforgeryToken" xml:space="preserve">
+    <value>Use Auto Validate Antiforgery Token</value>
```

Comment:
>Use Auto Validate Antiforgery Token [](start = 11, length = 35)

Going forward, let's use sentence casing for titles (like docs.microsoft.com guidelines):

Use antiforgery tokens in ASP.NET Core MVC controllers